streamer/src/recvmmsg.rs

```diff
@@ -187,9 +188,20 @@ mod tests {
     type TestConfig = (UdpSocket, SocketAddr, UdpSocket, SocketAddr);
 
     fn test_setup_reader_sender(ip_str: &str) -> io::Result<TestConfig> {
-        let reader = UdpSocket::bind(ip_str)?;
+        let sock_addr: SocketAddr = ip_str
+            .parse()
+            .map_err(|e| io::Error::new(io::ErrorKind::InvalidInput, e))?;
+        let reader = bind_to(
+            sock_addr.ip(),
+            /*port:*/ sock_addr.port(),
```

Comment:
Same as above - I think `sock_addr.port()` is telling enough that we don't need the `/*port*/` comment